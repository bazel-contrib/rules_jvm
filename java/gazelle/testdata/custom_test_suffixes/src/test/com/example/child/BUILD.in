# gazelle:java_test_file_suffixes FooTest.java,BarTest.java
