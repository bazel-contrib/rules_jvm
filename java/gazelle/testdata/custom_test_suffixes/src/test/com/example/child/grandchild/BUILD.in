# gazelle:java_test_file_suffixes Test.java
