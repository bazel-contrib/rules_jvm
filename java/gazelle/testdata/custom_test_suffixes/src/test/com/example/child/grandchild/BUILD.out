load("@contrib_rules_jvm//java:defs.bzl", "java_test_suite")

# gazelle:java_test_file_suffixes Test.java

java_test_suite(
    name = "grandchild",
    srcs = ["FooTest.java"],
)
